developer_guide.md

```diff
@@ -1,3 +1,60 @@
+
+## Directories Layout
+
+```sh
+$ tree -d -I 'vendor|bin|.git'
+├── build
+│   ├── images
+│   │   └── tf_operator
```

Comment:
What is in the images directory? Dockerfiles for images?

Reply:
Yes.